Makefile

```diff
@@ -36,14 +38,20 @@ SRC = $(shell find . -type f -name '*.go' -not -path "./vendor/*" -not -path "./
 
 all: build #check install
 
-build: fmt
-	operator-sdk build $(IMAGE_PATH)
+build: fmt operator-sdk
+	$(GOPATH)/bin/operator-sdk build $(IMAGE_PATH)
+
+operator-sdk:
+ifeq ("$(wildcard $(GOPATH)/bin/operator-sdk)","")
+	wget $(OPERATOR_SDK_URL) -O $(GOPATH)/bin/operator-sdk
```

Comment:
Should we add an OR after the wget that tells the admin to install operator-sdk themselves and retry if the command fails for whatever reason?

Reply:
this would be nice